lib/generators/authentication/templates/controllers/api/identity/email_verifications_controller.rb.tt

```diff
@@ -5,6 +5,8 @@ class Identity::EmailVerificationsController < ApplicationController
 
   def edit
     @user.update! verified: true
+
```

Comment:
little thing... can we remove this blank line ? 😁

Reply:
Done!